File: Atividade Extra 2/ControleDeAudio.ts
class ControleDeAudio {
    volume : number = 2;

    aumentarVolume (): void{
        if (this.volume <10){
            this.volume += 1
            }
        }
    
    diminuirVolume(): void {
        if (this.volume > 0){
            this.volume -= 1
        }
    }
    lerVolume(): number {
        return this.volume
    }
    }

let controle = new ControleDeAudio();
controle.aumentarVolume();
console.log(controle.lerVolume()); 
controle.diminuirVolume();
console.log(controle.lerVolume()); 

File: Atividade Extra 2/ControleDeAudio.java
public class ControleDeAudio {
    private int volume = 2;

    public void aumentarVolume() {
        if (volume < 10) {
            volume++;
        }
    }

    public void diminuirVolume() {
        if (volume > 0) {
            volume--;
        }
    }

    public int lerVolume() {
        return volume;
    }

    public static void main(String[] args) {
        ControleDeAudio controle = new ControleDeAudio();
        controle.aumentarVolume();
        System.out.println(controle.lerVolume());
        controle.diminuirVolume();
        System.out.println(controle.lerVolume());
    }
